addons/sys_signal/XEH_preInit.sqf

```diff
@@ -6,10 +6,14 @@ PREP_RECOMPILE_START;
 #include "XEH_PREP.hpp"
 PREP_RECOMPILE_END;
 
+// CBA Settings
+#include "initSettings.sqf"
+
 if (hasInterface) then {
     DGVAR(showSignalHint) = false;
     DGVAR(terrainScaling) = 1;
     DGVAR(omnidirectionalRadios) = 0;
+    DGVAR(signalModel) = SIGNAL_MODEL_LOS_MULTIPATH;
```

Comment:
Why is this necessary? It will be set with the setting registration.

Reply:
Never have a variable unitialised. It is just safer this way.

Reply:
But it's not uninitialized. If anything, you are changing the default from settings here.

I am not sure at which stage CBA does it, but you might be overwriting user set option.

Reply:
All other variables are initialised here: Omnidirectional radios, terrain loss scaling (which is set to 1 and later changed to 0.5 by CBA_settings). So it is the right place in my opinion to init all these. Testing shows that as well.

Reply:
Seems weird and unneeded. :thinking:

Reply:
Moved CBA settings after the initialisation of the variables.

Reply:
This makes a lot more sense now! :laughing: :+1: